sdk/communication/azure-communication-chat/env-test.properties

```diff
@@ -1,9 +1,14 @@
 trouter.client.version="99116/0.0.0.0/"
 trouter.max.registration.ttls="3600"
-platform="SPOOL"
+platform="Android"
 platform.ui.version="0.0.0.0"
 
 trouter.hostname="go.trouter-int.skype.net"
 trouter.registration.hostname.and.base.path="edge.skype.net/registrar/testenv"
 trouter.application.id="AcsAndroid_test"
 trouter.template.key="AcsAndroid_Chat_test_1.2"
+
+pushnotification.registrar.service.url="https://edge.skype.net/registrar/testenv/v2/registrations"
+pushnotification.application.id="AcsAndroid"
+pushnotification.template.key="AcsAndroid.AcsNotify_Chat_1.0"
```

Comment:
```suggestion
pushnotification.application.id="AcsAndroid_test"
pushnotification.template.key="AcsAndroid_test.AcsNotify_Chat_1.0"
```

Reply:
In INT PNH template for fcm push notification we don't put a "test" in appId/templateKey, which is a little inconsistency with what we have for trouter.

https://skype.visualstudio.com/SCC/_git/async_notification_templates?path=%2FTestEnv%2Ffcm%2FACSNotify%2FChat%2F1.0%2Fbase.json

We use this appId/templateKey because Calling does the same: using same appId/templateKey in both INT & PROD for FCM. We are making this part consist with Calling. https://skype.visualstudio.com/SCC/_git/async_notification_templates?path=%2FTestEnv%2Ffcm%2FACSNotify%2FCalling%2F2.0%2Fbase.json

Reply:
I can't access the repositories you shared but if you guys are planning on making a change to be consistent with Calling then that's fine by me.